cdap-cli/src/main/java/co/cask/cdap/cli/util/ArgumentParser.java

```diff
@@ -56,6 +56,8 @@
         String key = token.substring(0, firstEquals);
         String value = token.substring(firstEquals + 1, token.length());
         result.put(extractValue(key), extractValue(value));
+      } else {
+        throw new IllegalArgumentException(description + " must be of the form 'key1=val1 key2=val2'");
```

Comment:
Perhaps we would, but I kept it consistent with what the command help states for other commands:
```
  create dataset instance <dataset-type> <new-dataset-name> [<dataset-properties>] [<dataset-description>]
    Creates a dataset. '<dataset-properties>' is in the format 'key1=val1 key2=val2'.
```

I am all for changing it, but that would be a walk-through of all commands in CLI and beyind the scope if this fix.